### DISCUSSION TITLE
Skins folder missing

### DISCUSSION THREAD

**Discussion by sforsandeep** [ColorlibHQ/AdminLTE#3641] [Q&A] [answered]
Theme is simply awesome.. Love it. I was actually trying to do something with the blank template starter. But according to the documentation I checked the adminlte folder under node_modules, I was not able to locate the skins folder. Am I missing something

Thanks in advance

**Comment by danny007in** [comment, marked as answer]
what is adminlte version 
if it is v2 then its maintenance support is ended, you must upgrade to v3

**Reply by sforsandeep** [reply]
I specified admin-lte@latest when installing using npm. I checked the path and there is no skins folder under that path specified in starter page

**Reply by danny007in** [reply]
in adminlte v3 there is no skin folder
its built-in
check https://adminlte.io/docs/3.1/
